src/app/controllers/frontend/papers.ts

```diff
@@ -0,0 +1,148 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import * as DocumentTypes from '../../models/interfaces';
+import { APIOptions } from '../../../config/interfaces';
+import { PaperStats } from '../../../types';
+import { UNKNOWN } from '../../../config/default';
```

Comment:
What is UNKNOWN?

Reply:
If there are empty fields the value of `UNKNOWN` will be inserted instead, as a substitution. This is to prevent issues when querying the data. It was moved to `consts.ts`